src/services/orders.js

```diff
@@ -0,0 +1,149 @@
+const { Orders } = require('../models');
+const { OrderStatuses } = require('../models');
+const { Product } = require('../models');
+
+class OrdersService {
+  async createOrder (req, res) {
+    const orderStatus = await OrderStatuses.findOne({
+      name: 'In progress'
+    }).exec();
+    const order = new Orders(req.body);
+    order.status = orderStatus._id;
+    await order.save();
+    return res.json(true);
+  }
+
+  async getOrders (req, res) {
+    const { _id, status } = req.query;
+    const query = {};
+
+    if (_id) query._id = _id;
+    if (status) query.status = status;
+
+    const count = Orders.find(query).countDocuments().exec();
+    const orders = await Orders.find(query).exec();
+    return {
+      total_size: count,
+      list: orders ? res.json(orders) : res.json([])
+    };
+  }
+
+  async updateOrder (req, res) {
+    const order = await Orders.findOne({ _id: req.params.id }).exec();
```

Comment:
Move this method to orderStatuses service

Reply:
Fixed